src/Simulation/CsharpGeneration.Tests/EntryPointTests.fs

```diff
@@ -481,6 +481,9 @@ let ``Supports default custom simulator`` () =
     given ["--simulator"; typeof<QuantumSimulator>.FullName; "--use-h"; "false"] |> fails
 
 
+// TODO: Add tests for the "submit" command.
```

Comment:
:) 
What's the plan for tests? Are there any test planned on the runtime repo, or will this be tested as part of our service tests?

Reply:
My plan is to test all of the command-line option handling in the runtime repo using the `NothingMachine` as the target. For other quantum machines like IonQ, I think those will have to go in another repository where those machines are available by a direct reference.

Reply:
Tests are in #204.

Reply:
sounds good.